test/unit_test/components/gantry_test.dart

```diff
@@ -0,0 +1,264 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:grpc/grpc.dart';
+import 'package:viam_sdk/src/components/gantry/service.dart';
+import 'package:viam_sdk/src/gen/component/gantry/v1/gantry.pbgrpc.dart';
+import 'package:viam_sdk/src/resource/manager.dart';
+import 'package:viam_sdk/src/utils.dart';
+import 'package:viam_sdk/viam_sdk.dart';
```

Comment:
I'm torn about this one... 

For one I think you're right, import only what we need since this is a unit test.

On the other hand, the entry point into the SDK is the main `viam_sdk.dart` file, and importing that tests that our entry point is valid, and also that it contains the correct export (e.g. `gantry` in this case, which I actually had forgotten to add at first).

Reply:
Thats a good point.

would it be fair to write a test specifically for verifying we are exporting the expected files? I don't think we need to be too picky either way, how it is now would be fine and we can work under that assumption of we're testing consuming the package as well.